extension/data/styles/usernotes.css

```
@@ -93,11 +93,16 @@ body.mod-toolbox-rd.mod-toolbox-rd-new-modmail .tb-recents .tb-bracket-button {
     padding-top: 4px;
 }
 
+.mod-toolbox-rd .utagger-popup .utagger-types .utagger-type-list{
+    display: flex;
+    flex-wrap: wrap;
+}
+
```

```suggestion
.utagger-type-list > div {
  margin: 2px 2px 0 0;
}
```

Adds the margin @Geo1088 suggested